src/gui/systray.cpp

```diff
@@ -530,6 +531,13 @@ bool Systray::useNormalWindow() const
     return cfg.showMainDialogAsNormalWindow();
 }
 
+bool Systray::quickStyleProvidesTextFieldContextMenus() const
+{
+    static constexpr std::array styles{"org.kde.breeze", "org.kde.desktop"};
+    const auto style = QQuickStyle::name().toStdString();
+    return std::find(styles.cbegin(), styles.cend(), style);
```

Comment:
I get a build error on Windows here:

```
C:\Nextcloud\desktop\src\gui\systray.cpp(538): error C2440: 'return': cannot convert from '_InIt' to 'bool'
          with
          [
              _InIt=std::_Array_const_iterator<const char *,2>
          ]
  C:\Nextcloud\desktop\src\gui\systray.cpp(538): note: No user-defined-conversion operator available that can perform this conversion, or the operator cannot be called
```

this is enough to satisfy the cl compiler again, not sure if that's correct though:

```suggestion
    return std::find(styles.cbegin(), styles.cend(), style) != styles.cend();
```

Reply:
Thanks for the heads up, implemented